packages/changelog/src/index.ts

```diff
@@ -232,16 +240,20 @@ const applyChanges = ({
         Object.keys(changeTags).forEach((k) => (groupedChangesByTag[k] = []));
 
         // fill `deps` tag
-        const dependecies = Object.keys(
+        const dependencies = Object.keys(
           assembledChanges.releases[change.changes.name].changes
             .filter((c) => c.meta?.dependencies)
             // reduce to an object of keys to avoid duplication of deps
             .reduce((acc, c) => {
               c.meta!.dependencies.forEach((dep) => (acc[dep] = 1));
               return acc;
             }, {} as { [k: string]: any })
-        ).map((dep) => ({ summary: `Updated to latest \`${dep}\`` }));
-        groupedChangesByTag.deps.push(...dependecies);
+        ).map((dep) => ({
+          summary: getVersionFromApplied(dep, applied)
+            ? `Upgraded to \`${dep}@${getVersionFromApplied(dep, applied)}\``
+            : `Updated to latest \`${dep}\``,
+        }));
```

Comment:
Double invocation and wording consistency
```suggestion
        ).map((dep) => {
          const appliedVersion = getVersionFromApplied(dep, applied);
          return {
          	summary: appliedVersion 
            	? `Upgraded to \`${dep}@${appliedVersion}\``
            	: `Upgraded to latest \`${dep}\``,
            }
        });
```

Reply:
Aye, should have did this from the get. 👍